marimbabot_planning/src/planning.cpp

```diff
@@ -60,68 +60,94 @@ void Planning::go_to_home_position()
  *goal
  * @param  start_state
  * @param  goal_point
```

Comment:
please rm this line and add
@param left_mallet_goal_point
@param right_mallet_goal_poin

Reply:
Good point